apis/v1alpha1/gateway_types.go

```diff
@@ -649,6 +641,32 @@ const (
 	GatewayReasonAddressNotAssigned GatewayConditionReason = "AddressNotAssigned"
 )
 
+const (
+	// This condition is true when resources referenced by this Gateway exist.
+	// This covers any direct references to resources in the Gateway, including
+	// GatewayClassName and CertificateRef. This does not include selector
+	// references. This does not reflect the validity of the referenced
+	// resources, simply if the referenced resources exist.
```

Comment:
> simply if the referenced resources exist

Is there a word missing here? Maybe `simply validates if the referenced resources exist`?